### ISSUE TITLE
Predata for PRW dataset

### ISSUE THREAD

**Issue by lvyibing** [sjtuzq/person_search_gcn#2]
GCN's input is extracted from Joint Detection and Identification Feature Learning~(CVPR2017)'s article? You only release CUHK-SYSU's predata. Do PRW dataset need to be processed through that paper code and entered into Context Graph?

**Comment by sjtuzq** [comment]
Yes, GCN's input is extracted from that paper's code. And PRW dataset also needs to be processed to extract features which should be organized using the same format as CUHK-SYSU dataset.

**Comment by sjtuzq** [comment]
Yes, GCN's input is extracted from that paper's code. And PRW dataset also needs to be processed to extract features which should be organized using the same format as that of CUHK-SYSU dataset.

**Comment by lvyibing** [comment]
Can you provide the processing data of PRW dataset?  I can't debug the code of that article.

**Comment by sjtuzq** [comment]
Since we are preparing to further our work to a journal, I'm really sorry that we can not release all the code and data currently.

**Comment by bigAndSmall** [comment]
> Since we are preparing to further our work to a journal, I'm really sorry that we can not release all the code and data currently.

How's your work to journal going on? Can you provide us the processing data of the predata?

**Comment by bigAndSmall** [comment]
> Since we are preparing to further our work to a journal, I'm really sorry that we can not release all the code and data currently.

How's your work to journal going on? Can you provide us the processing data of the predata? @sjtuzq

**Comment by zqx951102** [comment]
完整的代码还可以提供一下吗？ 还有希望吗
